gr-analog/lib/fastnoise_source_X_impl.h.t

```diff
@@ -50,6 +50,7 @@ namespace gr {
       void set_type(noise_type_t type);
       void set_amplitude(float ampl);
       void generate();
+      const std::vector<@TYPE@>& samples() const { return d_samples; };
```

Comment:
Can we honestly *not* do that code in the header? I know it's less code here, but I really don't see any upside to this.

Reply:
In public headers definitely, in this case I can move everything out into implementation in a separate commit.

Reply:
well, are you planning to do that?

Reply:
also, I'm not sure if one-line functions on the same line are OK per our style guidelines, but if they are, we might want to change that.

Reply:
Can you comment the one-line functions in the `.clang-format` PR if it is applicable? I'm not opposed to changing this and move the implementation. But this won't happen tonight.

Reply:
Yeah... and I'm not a fan of one-line functions either :grin: